docs/references/configuration.rst

```diff
@@ -842,6 +842,22 @@ server-trace-header
 
   The header name used to trace HTTP requests. See :ref:`trace_header`.
 
+.. _server-timing-enabled:
+
+server-timing-enabled
+-------------------
```

Comment:
Sphinx is failing because of some missing `-` here.

Reply:
https://github.com/PostgREST/postgrest-docs/actions/runs/6959545989/job/18937755711?pr=700